tasks.py

```diff
@@ -73,16 +78,22 @@ def render_resource(resource):
 
 def render_contact(resource):
     contents = []
+
+    # append websites
```

Comment:
The newline does make sense to break it up though

```suggestion
```

Reply:
we love pep-8 formatting